feature-pack-build-maven-plugin/src/main/java/org/wildfly/build/plugin/FeaturePackBuildMojo.java

```diff
@@ -107,7 +107,7 @@
     @Override
     public void execute() throws MojoExecutionException, MojoFailureException {
         copyContents();
-        try (FileInputStream configStream = new FileInputStream(new File(configDir, configFile))) {
+        try (FileInputStream configStream = new FileInputStream(new File(project.getBasedir(), configFile))) {
```

Comment:
This shouldn't change and won't work if the default is left at `${basedir}/feature-pack-build.xml`. It would essentially result in `${basedir}${basedir}/feature-pack-build.xml` with the defaults.